build_runner/lib/src/package_graph/build_config_overrides.dart

```diff
@@ -20,6 +20,13 @@ Future<Map<String, BuildConfig>> findBuildConfigOverrides(
   await for (final file in configFiles) {
     if (file is File) {
       final packageName = p.basename(file.path).split('.').first;
+      if (!packageGraph.allPackages.containsKey(packageName)) {
```

Comment:
nit: consider checking for a null packageNode instead

Reply:
Done